tests/unit/init/weird_file_name_core.js

```diff
@@ -22,7 +22,7 @@ $( document ).on( "mobileinit", function() {
 	// one call to itself
 	$.mobile.initializePage = function() {
 
-		$.mobile.intializePage = origInitializePage;
+		$.mobile.initializePage = origInitializePage;
```

Comment:
This needs to be reverted, then i think every thing here is fine this corresponds to an option in the actual code.